build.sbt

```diff
@@ -56,14 +54,11 @@ lazy val root = (project in file("."))
     ScoverageKeys.coverageHighlighting := true,
     scalacOptions ++= Seq(
       "-feature",
-      "-rootdir",
-      baseDirectory.value.getCanonicalPath,
-      "-Wconf:cat=deprecation:ws,cat=feature:ws,cat=optimizer:ws,src=target/.*:s",
-      //"-Wconf:cat=other-match-analysis&src=app/services/.*:e", TODO enable this and address errors
-      //"-Wconf:cat=other-match-analysis&src=app/connectors/.*:e", TODO enable this and address errors
-      "-Wconf:cat=other-match-analysis&src=app/controllers/.*:e",
-      "-Wconf:cat=other-match-analysis&src=app/repositories/.*:e"
-      //"-Wconf:cat=unused&src=app/.*:e" TODO enable this and address errors
+      "-deprecation",
```

Comment:
Did we decide not to enable match exhaustiveness, or this enabled i.e `-Wconf:cat=other-match-analysis:e`